components/expenses/ExpenseAttachedFiles.js

```diff
@@ -21,16 +21,16 @@ const ExpenseAttachedFiles = ({ files, onRemove, showInvoice, collective, expens
                 onClick={downloadInvoice}
                 isDownloading={isLoading}
                 fileName={getExpenseInvoiceFilename(collective, expense)}
+                caption="Expense Invoice"
```

Comment:
Strings need to be internationalized, either with `FormattedMessage` or `intl.formatMessage`

Reply:
I've internationalized these descriptions within the `UploadedFilePreview`, i.e. the raw string is passed here and winds up within a `FormattedMessage`